doc/Soundness.md

```diff
@@ -63,10 +63,10 @@ However, in a few cases, it is useful to start with a mutable reference in order
 to enforce _who_ can make certain calls. For example, setting up buffers in the
 SPI driver is, for practical reasons, deferred until after construction but we
 would like to enforce that it can only be called by the platform initialization
-function (before `main` starts). This is enforced because all references after
-the platform is setup are immutable, and the `config_buffers` method takes an
-`&mut self` (_Note: it looks like this is not strictly necessary, so maybe not a
-big deal if we can't do this_).
+function (before the kernel loop starts). This is enforced because all
+references after the platform is setup are immutable, and the `config_buffers`
+method takes an `&mut self` (_Note: it looks like this is not strictly
+necessary, so maybe not a big deal if we can't do this_).
```

Comment:
IMO, we could just remove this paragraph. The SPI buffers are held in `TakeCell`s now anyway, and using `&mut self` as a way to enforce that buffers are only configured at boot is not a pattern we have really carried over to other capsules.